src/nacl/bindings/utils.py

```diff
@@ -39,3 +39,59 @@ def sodium_memcmp(inp1, inp2):
     eqC = lib.sodium_memcmp(buf1, buf2, ln) == 0
 
     return eqL and eqC
+
+
+def sodium_increment(inp):
+    """
+    Increment the value of a byte-sequence interpreted
+    as a little-endian big integer
+
+    :param inp: input bytes buffer
+    :type inp: bytes
+    :return: integer value of ``inp`` incremented by one
+    :rtype: bytes
+
+    """
+    ensure(isinstance(inp, bytes),
+           raising=exc.TypeError)
+
+    ln = len(inp)
+    buf = ffi.new("unsigned char []", ln)
+
+    ffi.memmove(buf, inp, ln)
+
+    lib.sodium_increment(buf, ln)
+
+    return ffi.buffer(buf, ln)[:]
+
+
+def sodium_add(a, b):
+    """
+    compute a modular addition (a + b) mod 2^(8*len) in constant time
+    for a given common length of their respective little-endian encoded
+    byte arrays.
+
+    :param a: input bytes buffer
+    :type a: bytes
+    :param b: input bytes buffer
+    :type b: bytes
+    :return: integer value of (``a`` + ``b``) mod 2^(8*len(``a``))
```

Comment:
Same

Reply:
OK, will try to think about a better wording of "a byte buffer containing the little-endian byte encoding of a integer whose value is..." if something better doesn't come up with you.